includes/reports/overview/data-provider/class-sensei-reports-overview-data-provider-courses.php

```diff
@@ -22,6 +22,13 @@ class Sensei_Reports_Overview_Data_Provider_Courses implements Sensei_Reports_Ov
 	 */
 	private $last_total_items = 0;
 
+	/**
+	 * Array of students ids, retrieved from last query.
+	 *
+	 * @var array Total number of items
+	 */
+	private $last_items_ids = [];
```

Comment:
[here](https://github.com/Automattic/sensei/pull/5091/commits/838fd8d96ac7ab448b364be708d2d820d821b309)

Reply:
Nice!

Just one small suggestion - I think it should be `item_ids` instead of `items_ids` because it's a compound noun. Compound nouns normally form the plural by adding `-s` to the last word of the compound.

Reply:
I change this here: https://github.com/Automattic/sensei/pull/5091/commits/c3cd4f15bf87d93f16b6d10084937dc45f75f631

But it is not relevant anymore since the changes are deprecated.